testing/btest/language/vector.zeek

```diff
@@ -229,4 +229,16 @@ event zeek_init()
 	test_case( "negative index", v24[-1] == 5 );
 	test_case( "negative index", v24[-3] == 3 );
 
+	# For a vector-of-vectors, += of an empty vector should append it as
+	# a single element, not all of its elements (= nothing gets appended).
+	local v25: vector of vector of count;
+	v25 += vector();
```

Comment:
A `vector()` constructor has an "unspecified" type (which is what the change to `Expr.cc` leverages), which means it's a vector-of-whatever's-needed. This works because it's empty, so there are no yield values to worry about not having the right type. Let me know if that answers your question ...